zen/xdg-toplevel-view.c

```diff
@@ -23,20 +23,7 @@ zn_xdg_toplevel_view_handle_wlr_surface_commit(
     return;
   }
 
-  if (!(self->base.resize_status.edges & (WLR_EDGE_LEFT | WLR_EDGE_TOP))) {
-    return;
-  }
-
-  struct wlr_surface* surface = data;
-  int dx = 0, dy = 0;
-  if (self->base.resize_status.edges & WLR_EDGE_LEFT) {
-    dx = surface->previous.width - surface->current.width;
-  }
-  if (self->base.resize_status.edges & WLR_EDGE_TOP) {
-    dy = surface->previous.height - surface->current.height;
-  }
-  zn_view_move(
-      &self->base, self->base.board, self->base.x + dx, self->base.y + dy);
+  wl_signal_emit(&self->base.events.surface_resized, data);
```

Comment:
I think this signal doesn't need to have an argument.

```suggestion
  wl_signal_emit(&self->base.events.surface_resized, NULL);
```